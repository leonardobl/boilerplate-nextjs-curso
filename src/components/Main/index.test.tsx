import { screen, render } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('Deve renderizar um h1', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /react avan√ßado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
