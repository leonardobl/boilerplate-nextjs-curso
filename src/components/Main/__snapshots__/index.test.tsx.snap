// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Deve renderizar um h1 1`] = `
<main>
  <h1>
    react avan√ßado
  </h1>
</main>
`
